modules/runner/src/test/java/org/apache/ignite/internal/configuration/storage/ConfigurationStorageTest.java

```diff
@@ -78,4 +78,12 @@ public void testReadAllLatest() {
 
         assertThat(latestData, willBe(anEmptyMap()));
     }
+
+    @Test
+    public void testRealConfig() {
+        Map<String, Serializable> data = Map.of("network.port", 3344,
+                "network.portRange", 5, "network.nodeFinder.netClusterNode", new String[] {"localhost:3344", "localhost:3345"});
+
+        storage.write(data, 0);
```

Comment:
Don't we need to check something after writing data?
What's up with the test name? What do you mean by "real config"?